qt/scientific_interfaces/Indirect/ConvFit.cpp

```diff
@@ -161,6 +161,8 @@ std::string ConvFit::fitTypeString() const {
   return fitType;
 }
 
+void ConvFit::updateAvailableFitTypes() {}
```

Comment:
I have hard time to understand why is this needed, can you explain these changes?

Reply:
IndirectFitAnalysisTab was calling a slot called updateAvailableFitTypes, however ConvFit, MSDFit and IqtFit (which inherit from this) didn't have an updateAvailableFitTypes slot. This wasn't a good way of doing it so thank you for picking this up. Seeing as JumpFit is the only one that needs to update the available fit types, I have changed this so a signal is emitted instead by the IndirectFitAnalysisTab.